course_info/static/course_info/js/directives/HuEditableInput.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+    var app = angular.module('CourseInfo');
+    app.directive('huEditableInput', huEditableInput);
+
+    function huEditableInput() {
+	return {
+            restrict: 'E',
+	    scope: {
+		editable: '<',  // if not provided, defaults to null/false
+		field: '@',
+		formValue: '=',
+		isLoading: '&',
+		label: '@',
+		maxlength: '@',
+		modelValue: '='
+	    },
+            templateUrl: 'partials/hu-editable-input.html',
```

Comment:
hmmm -- is there some reason this wouldn't have worked for me when I was trying it in the controller file? Seems like the same path and syntax....